lib/modules/apostrophe-tasks/index.js

```diff
@@ -27,9 +28,83 @@ module.exports = {
 
     self.tasks = {};
 
-    // Add a command line task to Apostrophe.
+    // For use when you wish to execute an Apostrophe command line task from your code and continue,
+    // without using the command line or using the `child_process` module.
     //
-    // The group name, by convention, should be the name of your module.
+    // Except for `name`, all arguments may be omitted.
+    //
+    // If you do not pass a callback, a promise is returned.
+    //
+    // Examples (assume `products` extends `apostrophe-pieces`):
+    //
+    // `self.apos.tasks.invoke('apostrophe-users:add', [ 'admin', 'admin' ]).then(function() { ... })`
```

Comment:
differentiate username from group in examples, can be confusing `['bob', 'admin']`